Jellyfin.Server/Migrations/Routines/DisableMetricsCollection.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using MediaBrowser.Common.Configuration;
+using MediaBrowser.Model.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Jellyfin.Server.Migrations.Routines
+{
+    /// <summary>
+    /// Disable metrics collections for all installations since it can be a security risk if not properly secured.
+    /// </summary>
+    internal class DisableMetricsCollection : IMigrationRoutine
+    {
+        /// <inheritdoc/>
+        public Guid Id => Guid.Parse("{4124C2CD-E939-4FFB-9BE9-9B311C413638}");
+
+        /// <inheritdoc/>
+        public string Name => "DisableMetricsCollection";
+
+        /// <inheritdoc/>
+        public void Perform(CoreAppHost host, ILogger logger)
+        {
+            // Set EnableMetrics to false since it can leak sensitive information if not properly secured
+            var metrics = host.ServerConfigurationManager.Configuration.EnableMetrics;
+            if (metrics)
+            {
+                logger.LogInformation("Disabling metrics collection during migration");
+                metrics = false;
+
+                host.ServerConfigurationManager.SaveConfiguration("false", metrics);
+            }
+        }
+    }
+}
```

Comment:
Why is this migration needed if this is a new configuration property?

Reply:
Guess I misunderstood why we made a migration for the transcode throttling?

Figured this was required to ensure it starts as off.

Reply:
No, that one was an existing setting that had a buggy behavior. The intent of that migration was to disable it for everyone moving forward (but not remove it in case some people still needed it and wanted to turn it back on)